example/src/screens/GraphPage.tsx

```diff
@@ -52,7 +59,23 @@ export function GraphPage() {
         enablePanGesture={enablePanGesture}
         enableFadeInMask={enableFadeInEffect}
         onGestureStart={() => hapticFeedback('impactLight')}
-        SelectionDot={SelectionDot}
+        SelectionDot={enableCustomSelectionDot ? SelectionDot : undefined}
+        range={
+          enableRange
+            ? {
+                x: {
+                  min: points[0]!.date,
+                  max: new Date(highestDate.getTime() + 30),
+                },
+                y: {
+                  min: -200,
+                  max: 200,
+                },
+              }
+            : undefined
+        }
```

Comment:
Please move that into a `useMemo`. This ternary is way too complex, do clean and easy to read `if`s.